### DISCUSSION TITLE
When using GFPGAN is it possible to just keep the final image in the samples folder instead of both?

### DISCUSSION THREAD

**Discussion by Wang-Thunder** [hlky/stable-diffusion-webui#349] [Q&A]
Updated repo today and noticed that there is a copy of the pre/post GFPGAN image in the samples folder. Is there a setting somewhere to make it like the previous webui and just store the corrected image instead of both?

Thanks! :)

**Comment by Telecino333** [comment]
Do the contrary of this thread here: https://github.com/sd-webui/stable-diffusion-webui/discussions/657

Switch Skip_Save to True
